PowerEditor/src/ScitillaComponent/FindReplaceDlg.h

```diff
@@ -401,7 +401,7 @@ private :
 	void updateCombo(int comboID);
 	void fillFindHistory();
     void fillComboHistory(int id, const std::vector<generic_string> & strings);
-	int saveComboHistory(int id, int maxcount, std::vector<generic_string> & strings);
+	int saveComboHistory(int id, int maxcount, std::vector<generic_string> & strings, bool saveEmpty = false);

```

Comment:
Why saveEmpty needs to be optional? And we should avoid paramter's default value - it can be a pitfall and make some bugs.